model/_global/bulk/operations.smithy

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+// TODO: Fill in API Reference URL
+@externalDocumentation(
+  "API Reference": ""
+)
+
+@vendorExtensions(
+  "x-operation-group": "bulk",
+  "x-version-added": "1.0"
+)
+@http(method: "POST", uri: "/_bulk")
+@documentation("Allows to perform multiple index/update/delete operations in a single request.")
+operation Bulk_0 {
+  input: Bulk_0_Input,
+  output: Bulk_Output
+}
+
+@vendorExtensions(
+  "x-operation-group": "bulk",
+  "x-version-added": "1.0"
+)
+@http(method: "PUT", uri: "/_bulk")
+@documentation("Allows to perform multiple index/update/delete operations in a single request.")
+operation Bulk_1 {
+  input: Bulk_1_Input,
+  output: Bulk_Output
+}
+
+@vendorExtensions(
+  "x-operation-group": "bulk",
+  "x-version-added": "1.0"
+)
+@http(method: "POST", uri: "/{index}/_bulk")
+@documentation("Allows to perform multiple index/update/delete operations in a single request.")
+operation Bulk_2 {
```

Comment:
I think you can achieve uniqueness while avoiding the numbering by following the naming template of some of the handwritten operations: `{method}{operation}With{pathParams}`
e.g:
```
PostBulk
PutBulk
PostBulkWithIndex
PutBulkWithIndex
```

Reply:
I saw that but there are actions with multiple endpoints using the same action like The [index](https://github.com/opensearch-project/opensearch-api-specification/pull/83/files#diff-dadd5d90ddef675d0e60edcf193aea5a5bc1565e603cf849993af22a1e058649) action where there are 2 POSTs.

Reply:
There are other actions that use the same verb twice

Reply:
I think it should just be something sort of like this pseudo code:
```
with = pathParams.len > 0 ? "With${pathParams.map(CamelCase).join("And")}" : ""
name = "${CamelCase(method)}${CamelCase(operationName)}${with}"
```

Reply:
The `WITH` suffix was a [bit more complicated](https://github.com/nhtruong/opensearch-smithy-spec-translator/commit/38867328699d7b845cdb4a9cb171a4b8827c76de#diff-4d29948f6a7d0e22ddb3eb9e4547957e17d90a43d5352bacd6775120cdcc6ff1R54-R60) but it's been solved. Thanks for the nudge! :D (Also that method would have been 3-4 times as long in JS or Python. That's why I love Ruby so much!)